cnd/src/storage/db/errors.rs

```diff
@@ -54,4 +54,4 @@ pub struct NoRefundIdentity;
 
 #[derive(thiserror::Error, Debug, Clone, Copy)]
 #[error("Order {0} is no longer open and can therefore not be cancelled")]
-pub struct NotOpen(pub OrderId);
+pub struct NotOpen(pub i32);
```

Comment:
This should stay as the order id, the PK will not be very useful in error messages.